stock_picking_declared_value/views/stock_picking_views.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2025 Nitrokey GmbH
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <!-- Add declared value fields to the stock.picking form view -->
+    <record id="view_picking_form_inherit_declared_value" model="ir.ui.view">
+        <field name="name">stock.picking.form.inherit.declared.value</field>
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.view_picking_form" />
+        <field name="arch" type="xml">
+            <!-- Add currency field (invisible) for monetary fields -->
+            <field name="origin" position="after">
+                <field name="currency_id" invisible="1" />
+                <field
+                    name="sale_id"
+                    readonly="1"
+                    attrs="{'invisible': [('sale_id', '=', False)]}"
+                />
+            </field>
+
+            <!-- Add declared value fields to the move lines -->
+            <xpath
+                expr="//field[@name='move_ids_without_package']/tree"
+                position="inside"
+            >
+                <field name="currency_id" invisible="1" />
+                <field name="sale_line_id" invisible="1" />
+                <field
+                    name="sale_unit_price"
+                    attrs="{'column_invisible': [('parent.sale_id', '=', False)]}"
```

Comment:
Still they will be set if if there is no `sale_id` set. Looks weird to hide them but still get an amount down the line.